forge-gui/res/cardsfolder/g/giant_regrowth.txt

```diff
@@ -0,0 +1,8 @@
+Name:Giant Regrowth
+ManaCost:1 G G
+Types:Instant
+A:SP$ ChangeZone | Cost$ 1 G G | Origin$ Graveyard | Destination$ Hand | TgtPrompt$ Select target card in your graveyard | ValidTgts$ Card.YouCtrl | SubAbility$ DBEffect | RememberObjects$ Targeted | SpellDescription$ Return target card from your graveyard. If it's a creature card, it perpetually gets +3/+3. to your hand.
```

Comment:
Card.YouOwn is a bit better for graveyard stuff

This SpellDesc doesn't match the Oracle